src/store/getters/state-getters.ts

```diff
@@ -115,3 +115,55 @@ export const isBusySaving = (state: any): boolean => {
 export const isApplicationValid = (state: any): boolean => {
   return (state.stateModel.defineCompanyStep.valid && state.stateModel.addPeopleAndRoleStep.valid)
 }
+
+/**
+ * Steps to be displayed in the stepper
+ */
+export const getSteps = (state: any, getters:any): Array<any> => {
+  const steps: Array<any> = [{
```

Comment:
There is a lot of overlap between the steps and the routes, and I thought this would the opportunity to combine them.

In particular, each route could specify (in meta) the "next" and "prev" steps, if any (else null).

What do you think about this, instead of having 2 structures with the same number of steps and step ids?

Reply:
Actually I dont see much overlap. The two common properties are 'step' and 'to'. 'to' can be removed from the steps as it can be derived from the routes based on the step number. But I am not in agreement with pusing many extra properties to meta of the route. Stepper needs additional properties like valid, enabled disabled etc and in my opinion it shouldnot go to the routes as routes is for the full application and it need not have any information related to the component validity or state.

Also previous and next steps are not needed as in this case you will go to the next higher step or next lower step only when you click the next or previous button. That is why it is a computed property now.

Reply:
OK for now then.